python/nnvm/frontend/coreml.py

```diff
@@ -269,6 +269,12 @@ def PaddingLayerParams(op, insym, symtab):
         raise NotImplementedError("Only constant padding is supported now.")
     return insym
 
+def PermuteLayerParams(op, insym, symtab):
+    axes = tuple(op.axis)
+    return _sym.transpose(insym, axes=axes)
+
+
```

Comment:
two blank space between functions and defs, instead of three

Reply:
fixed